persist/fs/read.go

```diff
@@ -260,7 +271,7 @@ func (r *reader) Read() (ts.ID, []byte, error) {
 
 	r.entriesRead++
 
-	return ts.BinaryID(entry.Id), data, nil
+	return ts.StringID(string(entry.Id)), data, nil
```

Comment:
Well this is only during bootstrap, once per ID.  Instead of on the write and read path which is once per datapoint.  

This was definitely a hack as I was unsure about the ownership of bytes created by the protobuf library so wanted to perform a copy here.  Instead of `StringID(...)` though I can introduce a `BinaryID(append(r.bytesPool.Get(len(entry.Id)), entry.Id...))`.